components/profile/ProfileDetail.vue

```diff
@@ -441,7 +441,6 @@ const { data: followers, refresh: refreshFollowers } = useAsyncData(
   () =>
     fetchFollowersOf(route.params.id as string, {
       limit: 3,
-      exclude: [accountId.value],
```

Comment:
won't this break something?

Reply:
I do not know why it excludes the login account before. IMO It should show all the followers/following members.

Reply:
Because there was an opened issue for that, 
Prob @exezbcz remebers

Reply:
@exezbcz Is there any special reason not to show the currently connected wallet address in the follower list?

Reply:
- https://github.com/kodadot/nft-gallery/issues/10242

Reply:
thanks for finding it out.